lib/r10k/git.rb

```diff
@@ -19,8 +20,12 @@ module Git
           :feature => :shellgit,
           :module  => R10K::Git::ShellGit,
         }
-      ],
-      [ :rugged,
+      ]
+    ]
+
+    unless R10K::Util::Platform.jruby?
+      @providers << [
```

Comment:
I don't think we need to gate this here, since the feature detection logic can look for the presence/absence of the gem (and since it can't be installed in JRuby it shouldn't be used.)